recipes/PyMuPdf/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "PyMuPdf" %}
+{% set version = "1.17.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/e8/a8/c5774138aa34f1fe24d9efeb1506f148bb55c52583bf44b0f9d48cbcd59e/PyMuPDF-1.17.3.tar.gz
```

Comment:
Is there a reason that `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz` doesn't work?

Reply:
You can try it yourself, it does not work. Probably a lowercase/uppercase issue.